config/module.config.php

```diff
@@ -6,6 +6,10 @@
         ],
     ],
     'phpab' => [
+        'analytics' => [
+            'collector' => null,
```

Comment:
Once we have the DataCollector\Generic we could set it as default.
Renderer is very user specific, instead.